cfme/automate/dialogs/dialog_element.py

```diff
@@ -181,9 +182,17 @@ def create(self, element_data):
             view.fill(element)
             view.ele_save_button.click()
         view.save_button.click()
-        view.flash.assert_no_error()
-        view.flash.assert_message('{} was saved'.
-                                  format(self.parent.tab.dialog.label))
+        error_message = version.pick({
```

Comment:
**REQUIRED**
```python
if self.appliance.version < "5.9":
    error_message = ...
else:
    error_message = ...
```

Reply:
I'm not seeing the benefit of including the expected flash message text in the `AssertionError` - if anything I would think the content of the UI flash message should be included, so whatever called create() that resulted in a flash error message could assert against the content of the exception.